data/common.yaml

```diff
@@ -4,4 +4,5 @@ kibana::config: {}
 kibana::manage_repo: true
 kibana::oss: false
 kibana::package_source: ~
+kibana::plugindir: ~
```

Comment:
With the default value in init.pp, no need to add it in hiera, so please also remove this line.